yt_dlp/extractor/bilibili.py

```diff
@@ -502,39 +503,148 @@ def _real_extract(self, url):
             season_info.get('bangumi_title'), season_info.get('evaluate'))
 
 
-class BilibiliChannelIE(InfoExtractor):
-    _VALID_URL = r'https?://space.bilibili\.com/(?P<id>\d+)'
-    _API_URL = "https://api.bilibili.com/x/space/arc/search?mid=%s&pn=%d&jsonp=jsonp"
+class BilibiliSpaceBaseIE(InfoExtractor):
+    def __init__(self, downloader=None):
+        InfoExtractor.__init__(self, downloader)
+        self.page_idx_start = 1
+
+    def _fetch_page(self, playlist_id, page_idx, **kwargs):
+        raise NotImplementedError('This method must be implemented by subclasses')
+
+    def _get_metadata(self, page_data, **kwargs):
```

Comment:
You can remove `**kwargs` from all `_get_metadata` functions

Reply:
ok